algorithms/ets.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+class ETS:
+    def __init__(self):
+        self.filename = "ets"
+
+    def router_command(self, router, interface):
+        router.cmd(f'tc qdisc del dev {interface} root')
+        router.cmd(f'tc qdisc add dev {interface} root ets')
```

Comment:
I think here the algor ETS requires to specify class. Can refer to the following code:
```
r.cmd('tc qdisc add dev r-eth6 root handle 1: htb default 10')
r.cmd('tc class add dev r-eth6 parent 1: classid 1:1 htb rate 10mbit')
r.cmd('tc class add dev r-eth6 parent 1: classid 1:2 htb rate 100mbit')
r.cmd('iptables --append FORWARD --table mangle --protocol tcp --dport 5200 --jump MARK --set-mark 2')
r.cmd('tc filter add dev r-eth6 parent 1:1 protocol ip handle 2 fw classid 1:2')
```